GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/SpaceViewModel.cs

```diff
@@ -0,0 +1,61 @@
+﻿using GalaxyZooTouchTable.Models;
+using PanoptesNetClient;
+using PanoptesNetClient.Models;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Threading.Tasks;
+
+namespace GalaxyZooTouchTable.ViewModels
+{
+    public class SpaceViewModel : INotifyPropertyChanged
+    {
+        public List<TableSubject> CurrentGalaxies { get; set; } = new List<TableSubject>();
+        public double RA { get; set; } = 250.2;
+        public double DEC { get; set; } = 35.1;
+
+        private string _spaceCutoutUrl;
+        public string SpaceCutoutUrl
+        {
+            get { return _spaceCutoutUrl; }
+            set
+            {
+                _spaceCutoutUrl = value;
+            }
+        }
+
+        public SpaceViewModel()
+        {
+            GetSpaceCutout();
+            GetSubjectsAsync();
+        }
+
+        private void GetSpaceCutout()
```

Comment:
Would it make more sense to just move the content of this function inline with the constructor?

Seems overly verbose to have a whole function for just one line of code

Reply:
As the centerpiece is built out, this method will have to be called whenever the space navigation changes to a different tile. This method will return a different image as the RA/DEC are changed. Note the string interpolation in this method.